### ISSUE TITLE
a warning

### ISSUE THREAD

**Issue by zhenhao-huang** [Strawberry-Eat-Mango/PCT_Pytorch#21]
UserWarning: Detected call of `lr_scheduler.step()` before `optimizer.step()`. In PyTorch 1.1.0 and later, you should call them in the opposite order: `optimizer.step()` before `lr_scheduler.step()`.  Failure to do this will result in PyTorch skipping the first value of the learning rate schedule. See more details at https://pytorch.org/docs/stable/optim.html#how-to-adjust-learning-rate "https://pytorch.org/docs/stable/optim.html#how-to-adjust-learning-rate", UserWarning)
Is it ok?

**Comment by 294coder** [comment]
pytorch warning, which means you should put "optimizer.step()" ahead of "lr_scheduler.step()" in your training phase.

**Comment by zhenhao-huang** [comment]
In the `main.py` script, I put `scheduler.step()` after the `train_loader` loop, and the warning disappeared.
